packages/ove-asset-manager/src/OVE.Service.AssetManager/Views/Shared/Error.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@model OVE.Service.AssetManager.Models.ErrorViewModel

```

Comment:
Please can we update the Error page in order to provide a useful error for a production user

Reply:
This page only catches uncaught global exceptions which will make limited sense except to a developer. We could enable showing these exceptions on this page but it is not recommened: 

Swapping to Development environment will display more detailed information about the error that occurred.
Development environment should not be enabled in deployed applications as it can result in sensitive information from exceptions being displayed to end users.